### PR TITLE
Feature/97 config file database path part1

### PR DESCRIPTION
**Pull request by drekoumis**
This code need to test it but you have to install toml in venv. Work perfect for sqlite. We have test and work with postgresql. Maybe s better the def db_conf_init(): function to move in  panoptes/database.py. i am happy to see your opinion

To install the latest release on PyPI, simply run:

`pip install toml`

Or to install the latest development version, run:

```
git clone https://github.com/uiri/toml.git
cd toml
python setup.py install
```

### CODE REVIEW

**Review by fgypas — CHANGES_REQUESTED** [review]
@drekoumis Can you please update the requirements and setup.py with `toml` to pass the tests? Also, I don't have experience with `toml`, but what is the advantage over `yaml`?

**Comment by drekoumis** [comment]
i can't fine any way to use engine_from_config and config file with separate Parameters  drivername, username,password ,database password, host, port, database,query without any check if is correct. If you wish to create a simple file with this parameters without any check and guidelines i can.

**Comment by drekoumis** [comment]
> 
> 
> @drekoumis Can you please update the requirements and setup.py with `toml` to pass the tests? Also, I don't have experience with `toml`, but what is the advantage over `yaml`?

https://www.barenakedcoder.com/blog/2020/03/config-files-ini-xml-json-yaml-toml/

**Comment by agardelakos** [comment]
Raised this feature request [feature_125]( https://github.com/panoptes-organization/panoptes/issues/125) as the result of the work on this PR.

File: panoptes/db.config [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{"Database":
```
Comment (agardelakos) [inline panoptes/db.config:1]: For usability reasons, I would very much prefer to have a clear indication on the file format that we use. Something like db_config.json....

Comment (drekoumis) [reply panoptes/db.config:1]: done file name change to db_config.json
done format

**Pull request closed without merging**